tests/actions/ReportPreviewActionUtilsTest.ts

```diff
@@ -58,6 +58,19 @@ describe('getReportPreviewAction', () => {
         await Onyx.set(ONYXKEYS.PERSONAL_DETAILS_LIST, {[CURRENT_USER_ACCOUNT_ID]: PERSONAL_DETAILS});
     });
 
+    it('isAddExpenseAction should return true for report preview with no transactions', async () => {
```

Comment:
Could we also add a case for it returning false if the report has a transaction?